bigbluebutton/greenlight
# CVE-2015-9284: follow-up for restoring single authentication method and LDAP

Closes #3374 





## Description

A new issue arises when there is only one authentication method. Since that is done with internal redirects, and redirects can not be done using POST, it needs to be tailored though a form.

Additionally, because all the actions triggered by links and buttons are tied to post, LDAP, which was not meant to respond to post requests simply failed. 

## Testing Steps

Case 1. Deploy GL with one single external authentication method. Sign In, should work as before.

Case 2. Either with one or multiple methods, enable LDAP as one of them, Instead of a 404, the user should see the sign in form again.

## Screenshots (if appropriate):
